src/NDSparseData.jl

```diff
@@ -209,6 +209,7 @@ function _getindex(t::NDSparse, idxs)
     end
     I = t.indexes
     lI = length(I)
+    map(sort!, idxs)
```

Comment:
I'm worried about the performance of this. Might be better to check `issorted` and throw an error instead? This also won't support immutable indexes, such as ranges or `:`.